ui/config-overrides.js

```diff
@@ -16,6 +16,11 @@ module.exports = {
             environmentFilePath
         )
     );
+
+    // removes the last minimizer CssMinimizerPlugin() which has a bug
+    // see https://github.com/all-of-us/workbench/pull/6753#issuecomment-1150407055
+    config.optimization.minimizer = config.optimization.minimizer.filter(m => m.options.minimizer.options.parse?.ecma);
```

Comment:
Thanks, this is definitely an improvement. To prevent this from regressing I would suggest asserting that this removes exactly one element. e.g.

```
const minimizersRemoved = originalMinimizerLength - config.optimization.minimizer.length;
if (minimizersRemoved !== 1) {
  throw new Error(`expected to remove exactly one minimizer, actually removed {minimizersRemoved}`);
}
```

Reply:
good idea, added